service-base/src/main/java/org/eclipse/hono/service/credentials/CompleteBaseCredentialsService.java

```diff
@@ -161,6 +189,15 @@ protected int getMaxBcryptIterations() {
         return DEFAULT_MAX_BCRYPT_ITERATIONS;
     }
 
+    /**
+     * Gets the hash function to be used to hash plaintext passwords.
+     * 
+     * @return The name of the hash function.
+     */
+    protected String getHashFunction() {
```

Comment:
is this method supposed to overridden? If so, then the JavaDoc should state so and indicate why a user would want to do so. It would also be helpful to know, which function names are actually supported.

Reply:
Added JavaDoc.